src/main/telemetry/mavlink.c

```diff
@@ -82,6 +82,66 @@
 #define TELEMETRY_MAVLINK_MAXRATE       50
 #define TELEMETRY_MAVLINK_DELAY         ((1000 * 1000) / TELEMETRY_MAVLINK_MAXRATE)
 
+
+/** @brief A mapping of plane flight modes for custom_mode field of heartbeat. */
+#ifndef HAVE_ENUM_PLANE_MODE
+#define HAVE_ENUM_PLANE_MODE
+typedef enum PLANE_MODE
```

Comment:
Please rename this to i.e. `APM_PLANE_MODE` to indicate that it's not INAV's native enum